jscomp/core/js_of_lam_block.ml

```diff
@@ -56,8 +56,10 @@ let field (field_info : Lam_compat.field_dbg_info) e i =
   | Fld_record_inline comment
   | Fld_record_extension comment
 #end
-  | Fld_record comment
-  | Fld_module comment
+| Fld_record comment when comment <> "contents" ->
+  E.dot e comment
```

Comment:
do we need treat ref type speically? we can change the definition of `ref` in pervasives